crates/astria-sequencer/src/bridge/storage/values/rollup_id.rs

```diff
@@ -0,0 +1,60 @@
+use std::{
+    borrow::Cow,
+    fmt::{
+        self,
+        Display,
+        Formatter,
+    },
+};
+
+use astria_core::primitive::v1::RollupId as DomainRollupId;
+use astria_eyre::eyre::bail;
+use borsh::{
+    BorshDeserialize,
+    BorshSerialize,
+};
+use telemetry::display::base64;
+
+use super::{
+    Value,
+    ValueImpl,
+};
+
+#[derive(Debug, BorshSerialize, BorshDeserialize)]
+pub(in crate::bridge) struct RollupId<'a>(Cow<'a, [u8; 32]>);
+
+impl<'a> Display for RollupId<'a> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        base64(self.0.as_slice()).fmt(f)
```

Comment:
Should assert this matches the formatting of the domain type `RollupId`

Reply:
As discussed offline, we can skip that since we're using Debug formatting.